src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.cs

```diff
@@ -167,16 +167,26 @@ public void Dispose()
                         Assert.True(Process.WaitForExit(Options.TimeOut),
                             $"Timed out after {Options.TimeOut}ms waiting for remote process {Process.Id}");
 
+                        if (File.Exists(Options.ExceptionFile))
+                        {
+                            Assert.True(false, File.ReadAllText(Options.ExceptionFile));
```

Comment:
Instead of doing an Assert.True(false) which will add an Assert failure saying: "Expected: true, Actual: false" why we don't add like a RemoteExecutionFailureException or something like that, that inherits from XunitException which has already a ToString() to print stack traces for test failures correctly and an inner exception. Then we can throw that exception here instead.

https://github.com/xunit/assert.xunit/blob/ae98601b8ed4758a5c7880b087f08a8ec06e1c4c/Sdk/Exceptions/XunitException.cs